epub33/a11y/index.html

```diff
@@ -1204,64 +1204,76 @@ <h4>Evaluator Information</h4>
 					<aside class="example">
 						<p>The following example shows a credential. The <code>refines</code> attribute associates the
 							credential with the certifier.</p>
-						<pre>&lt;meta
-    property="dcterms:conformsTo"
-    id="conf">
-   EPUB-A11Y-11_WCAG-21-AA
-&lt;/meta>
-&lt;meta
-    property="a11y:certifiedBy"
-    refines="#conf"
-    id="certifier">
-   EPUB Accessibility Evaluator
-&lt;/meta>
-&lt;meta
-    property="a11y:certifierCredential"
-    refines="#certifier">
-   A+ Accessibility Rating
-&lt;/meta></pre>
+						<pre>&lt;metadata …>
```

Comment:
Shouldn't there be a title for this example, too?

Reply:
Yup, I must have accidentally jumped past it.